### ISSUE TITLE
English Logo Displaying Incorrectly on Title Screen

### ISSUE THREAD

**Issue by Gamma-Vector** [07th-mod/umineko-question#103]
![logoproblem](https://user-images.githubusercontent.com/41348395/42849623-f1032698-89f1-11e8-8d1c-824fb1c0eb4c.PNG)
I'm about 99.9% certain this isn't how things are supposed to look.

I backed up the Japanese logo before installing the recent update, and noticed that the images are formatted differently. The Japanese logo is just one thing, with transparency etc.
![logo_jp](https://user-images.githubusercontent.com/41348395/42849713-72fcc280-89f2-11e8-85cd-74ce48194ce6.png) 
The English one appears to be...doubled? I'm honestly not sure what's going on here, but it looks to me like it's suppose to display both halves of the image overlayed on each other. Or something.
![logo](https://user-images.githubusercontent.com/41348395/42849709-70472116-89f2-11e8-9e0b-06344ee48c75.png)

In any case, it's not doing that. It's displaying the logo sortof centered on the left-hand side of the logo image file, and you can just barely see the right-hand side in my screenshot. I tried redownloading the exe and utf files as well, but that didn't fix the problem.

It looks to me like someone either forgot how the logo display was handled and incorrectly formatted the English image, or something was meant to be changed in how the logo display was handled, but that change didn't get implemented.

**Comment by drojf** [comment]
By popular demand I added update instructions to the wiki: https://github.com/07th-mod/guide/wiki/Umineko-Part-1---Voice-and-Graphics-Patch#updating-from-older-versions

**Comment by Gamma-Vector** [comment]
I did update the script/utf file, and I did get the new option about the PS3 movies when I started the game the next time. But for some reason, the logo is still displaying incorrectly for me. (Weirdly enough, it's just the title screen - the "transition screens" inside the chapter which show the logo are displaying correctly.) 

I'm going to try deleting everything (after moving my save files, of course), reinstalling via steam, and then using the installer to reinstall the patch. I'll let you know how it goes. Thank you very much for the quick response.

**Comment by drojf** [comment]
hmm...maybe let me review my changes, I could have screwed something up. I think you did everything correctly. Can you send me your script file? then I can directly check you have the right one (just zip it and drag it onto the git comment box).

The fact that you get that funky image to show at all means that you have the correct assets.

Also (incase it makes any difference) you're running windows yea? not mac/linux?

**Comment by Gamma-Vector** [comment]
Sorry for the late response. I didn't get a chance to go through with my delete-everything-and-then-resurrect plan until today, but it worked. A clean install via Steam, then using the installer program to download/install the patch has everything working correctly.

For reference, I was previously working my way through episode 4 on Windows with the Steam version of the game which had been patched with the installer program. I had updates via Steam disabled, and honestly I wouldn't be surprised if my base game was out of date for that reason, as I literally never paid attention to whether or not there was an update available. I don't know if that might have caused the problem, but my base game was entirely reinstalled today, so it's currently up to date. I haven't had a chance to play through anything to see if there are any new issues, but the logo display problem is fixed.

When the recent update to the patch came out, I wanted it primarily because I wanted the ability to choose which opening movies to use (I played the Witch Hunt translations back in the day and got weirdly attached to a few things). I downloaded the Update V5 zip and the new script/utf file. Unzipped the update manually, renamed 0.utf to 0.u, copied the script in, and started up the game. That's when I got the issue described in the OP. I thought I must have screwed something up, so I deleted the script file entirely and redownloaded a new copy, renamed it, etc. Still screwy. So then I deleted the exe file entirely, downloaded the one linked on the guide page, and tried that. Still a problem. That's when I posted the issue here.

I hope that helps.

**Comment by drojf** [comment]
Honestly I think you did everything correctly. The only step I can imagine you did wrong is, when installing the update zip, you extracted to a subfolder instead of the game folder (or your extractor program generated a subfolder when there shouldn't be one).

> I wanted it primarily because I wanted the ability to choose which opening movies to use (I played the Witch Hunt translations back in the day and got weirdly attached to a few things). 

I hate to disappoint you but the change we made only affects chapters 1-3  - we only made the change to reduce spoilers so we didn't change the chapter 4 movie because it was fairly clean of spoilers :S. 

I'll close this issue now, but if you have any problems with the answer arc etc. feel free to make another issue or talk on discord if it's something small. Or reopen this issue if you still have some logo issues.
